app/views/maintenances/index.html.haml

```diff
@@ -1,4 +1,14 @@
+=link_to t('add_new'), new_maintenance_path(@maintenance), class: 'btn btn-primary'
+
 .autopagerize_page_element
-  - @maintenances.each do |maintenance|
-    %p= link_to maintenance.name, maintenance
-= paginate @maintenances
+  %table.table.table-hover
+    %thead
+      %tr
+        %th{scope: 'col'}= Maintenance.model_name.human
+        %th{scope: 'col'}= Car.model_name.human
+        %th{scope: 'col', colspan: '2'}
+
+      - @maintenances.each do |maintenance|
```

Comment:
After changes you made you don't need the line:
```
- @maintenances.each do |maintenance|
```
That's one of the profit of change